src/components/StackNavigator.js

```diff
@@ -0,0 +1,31 @@
+import { createStackNavigator } from 'react-navigation';
+
+import { LoginScreen, ProductDetailsScreen, ProductsListScreen } from './';
+
+export const StackNavigatorRoutes = createStackNavigator(
+    {
+        Login: {
```

Comment:
Here you can also define the screens names as constants, and use them in all navigation code:
```
export const SCREEN_NAMES = {
  LOGIN: 'LOGIN',
  ...other screens names
}

createStackNavigator({
    [SCREEN_NAMES.LOGIN]: {
        screen: LoginScreen,
        navigationOptions: {
            title: "Login"
        }
    }
 ...otherConfigs,
})

```